deploy/template/tutorial-web-app.yml

```diff
@@ -77,6 +77,24 @@ objects:
             protocol: TCP
     triggers:
       - type: ConfigChange
+- apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
```

Comment:
The operator won't be able to create this resource without adding it to the role (https://github.com/integr8ly/tutorial-web-app-operator/blob/master/deploy/rbac.yaml). An example can be found here: https://github.com/integr8ly/keycloak-operator/blob/master/deploy/rbac.yaml#L51

Reply:
Thanks for that @pb82 I will looking add it to the rbac.yaml for this operator.

Reply:
I think this sdk version used here is a bit old and we will need to explicitly  add this kind into the operator or update the sdk version.

Reply:
@pb82 and @odra I have updated the rbac.yaml to include the new service monitor.

Reply:
@Nanyte25, @pb82  and I were talking and adding the ServiceMonitor object in the webapp template might require some operator code changes because it needs to "be aware" of the ServiceMonitor monitor kind and api group.

We were wondering if we could add it in a file of its own in the same folder and then apply this monitoring resource file from the installer for now.

Reply:
@odra I am open to any advice here whatever course of action here you deem appropriate. I will go with.

Reply:
@pb82 and @odra  I will revert the chnage and create a separate servicemonitor.yaml file in the same directory.

Reply:
@odra can you take a look at the changes